integrations/azure-devops/azure_devops/client/azure_devops_client.py

```diff
@@ -102,10 +122,8 @@ async def generate_repository_policies(
             for repo in repos:
                 params = {
                     "repositoryId": repo["id"],
+                    "refName": repo["defaultBranch"],
```

Comment:
Make sure to not overwrite things in the main branch, I pushed this code today and the logic is being overwritten here